### PR TITLE
[Hub Generated] Review request for Microsoft.Capacity to add version preview/2019-04-01

### PR DESCRIPTION
**Pull request by gautam714**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/gautam714/azure-rest-api-specs/dev-reservations-Microsoft.Capacity-2019-04-01_includeterm...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by nschonni** [comment]
Model validation issues fixed in #6732. If you pull those fixes in here, let me know an I'll close that one

**Comment by gautam714** [comment]
> Model validation issues fixed in #6732. If you pull those fixes in here, let me know an I'll close that one

I have pulled the fix. All validations are passing now. Could you please approve and merge?

**Comment by nschonni** [comment]
Sorry, i'm not an MS employee. I think there are instructions on the PR template for escalating internally

**Comment by daviwil** [comment]
@openapibot sdkautomation rebuild

**Comment by Juliehzl** [comment]
@NullMDR Can you have a look at SDK problem?

**Comment by NullMDR** [comment]
@openapibot sdkautomation rebuild

**Comment by NullMDR** [comment]
@Juliehzl This should be a bug in sdk automation. I'll check it later.

**Comment by NullMDR** [comment]
@openapibot sdkautomation rebuild

**Comment by gautam714** [comment]
Can somebody please take a look from ARM side? @Juliehzl : please assign someone.

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json [hunk 1]
```diff
@@ -1012,6 +1012,9 @@
         },
         "renewProperties": {
           "$ref": "#/definitions/RenewPropertiesResponse"
+        },
+        "term": {
+          "$ref": "#/definitions/ReservationTerm"
```
Comment (majastrz) [inline specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]: >ReservationTerm [](start = 33, length = 15)

You are adding a new property non-readonly property to a stable API version. This is considered a breaking change. You should introduce this change in a new API version.

**Review by majastrz — CHANGES_REQUESTED** [review]
You are adding a new property non-readonly property to a stable API version. This is considered a breaking change. You should introduce this change in a new API version.

**Comment by majastrz** [comment]
> You are adding a new property non-readonly property to a stable API version. This is considered a breaking change. You should introduce this change in a new API version.

When I was tracing usages through in the Swagger I mistook response for the body of a PUT, which led to me to conclude that it wasn’t read-only. This was also confirmed by the RP owners.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Review by Juliehzl — APPROVED** [review]

**Pull request merged**